spec/unit/knife/core/windows_bootstrap_context_spec.rb

```diff
@@ -82,12 +75,9 @@
   end
 
   describe "validation_key" do
-    before do
-      bootstrap_context.instance_variable_set(:@chef_config, Mash.new(validation_key: "C:\\chef\\key.pem"))
-    end
+    let(:chef_config) { { validation_key: "C:\\chef\\key.pem" } }
 
     it "should return false if validation_key does not exist" do
-      allow(::File).to receive(:expand_path).with("C:\\chef\\key.pem").and_call_original
```

Comment:
`allow(...).to receive(...).and_call_original` is essentially a no-op.

Reply:
lol, yeah.

Reply:
particularly with "allow" and not "expect", that's just silly